docs/pages/get-started/start-developing.mdx

```diff
@@ -86,7 +86,7 @@ Expo Go is configured by default to automatically reload the app whenever a file
 
 ## File structure
 
-Below, you can get familiar with the default project's file structure:
+Below, you can get familiar with the default project's file structure. This is one of many possible ways to structure your project, and the only enforced convention is usage of the **app** directory for routes with Expo Router. Feel free to rename directories and files and structure your project however you like.
 
 <ProjectStructure />
 
```

Comment:
```suggestion

<br />

> Read more about why [non-route files don't belong in the **app** directory](/router/create-pages/#non-route-files) when using Expo Router.
```

Let's also add this callout that links to the section where non-route files recommended structure is explained.